src/test/java/easytests/core/entities/QuizEntityTest.java

```diff
@@ -2,54 +2,43 @@
 
 import easytests.core.models.QuizModelInterface;
 import easytests.core.models.empty.IssueModelEmpty;
+import easytests.support.QuizzesSupport;
 import org.junit.Assert;
 import org.junit.Ignore;
 import org.junit.Test;
 import org.meanbean.test.BeanTester;
+import org.meanbean.test.ConfigurationBuilder;
 import org.meanbean.test.EqualsMethodTester;
 import org.meanbean.test.HashCodeMethodTester;
 import org.mockito.Mockito;
 
 import java.time.LocalDateTime;
 
 /**
- * @author fortyways
+ * @author miron97
  */
-public class QuizEntityTest {
-    @Ignore
+public class QuizEntityTest extends AbstractEntityTest {
```

Comment:
You remove testHashCode and testEquals because they fails? If so, I recommend to:
 - do not modify  AbstractEntityTest
 - add specific testCommon() class that will include: its own configuration inside, and test bean call also.